.github/workflows/static.yml

```diff
@@ -31,13 +31,24 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
       - name: Setup Pages
         uses: actions/configure-pages@v4
+
+      - name: Setup config.json
+        run: |
+          cd www/
+          sed -i "s;%GIT_VERSION%;$(git describe --tags);" index.html index.js
```

Comment:
The %GIT_VERSION% is only in `index.js`. There is no need to replace it in `index.html`.